dash-spv-ffi/FFI_API.md

```
@@ -33,77 +31,46 @@ Functions: 4
 
 ### Configuration
 
-Functions: 25
+Functions: 16
 
 | Function | Description | Module |
 |----------|-------------|--------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)